package.json

```diff
@@ -52,7 +52,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "format": "eslint . --fix",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "docs": "echo docs stub script"
```

Comment:
```suggestion
    "docs": "echo NOT IMPLEMENTED"
```

Reply:
Note: I think you have the wrong github email address.  If I choose to commit a suggestion from you,  the CLA check fails.    I had to update my author email to anon one: 4603341+greghutch@users.noreply.github.com before the CLA check stopped failing.   Maybe.

Reply:
Amended change with suggested text.